content/general/concepts/cicd-pipeline/cicd-pipeline.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: 'CICD Pipeline'
+Description: 'The CICD pipeline automates the software/application delivery process.'
+Subjects: 
+  - 'Developer Tools'
+  - 'Mobile Development'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Developer Tools'
+  - 'Development'
+CatalogContent:
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/back-end-engineer-career-path'
+---
+
+The **CICD Pipeline** is the combination of continuous integration, delivery and deployment of software development and application deployment. The four main steps of the pipeline are: Source, Build, Test and Deploy. 
+
+- **CI**: CI stands for Continuous Integration. This is the automation process for Developers.
+- **CD**: The first CD stands for continuous delivery. This is the automated delivery of code to the testing and development steps of the pipeline. These environments are usually in a repository like [Github](https://www.github.com)
+- **The other CD**: The second CD stands for continuous deployment. When code passes the automated tests, it is automatically placed in production, which is then used for automatic deployment. 
+
+## Steps of the CICD Pipeline
+
+There are four main steps to the CICD Pipeline: 
+1. Source
+2. Build
+3. Test
+4. Deploy
```

Comment:
Rather than just listing these out, let's go ahead and define them in your own words of course. I recommend looking into: https://circleci.com/docs/about-circleci/